package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "cloudevents",
-  "version": "7.0.0",
+  "version": "7.0.1",
```

Comment:
This should stay at 7.0.0
```suggestion
  "version": "7.0.0",
```

Reply:
I've changed it back

Reply:
@pieter-pon thanks - you also should revert package-lock.json.

Reply:
@lance Also revert the package-lock, should I update the lock file that only the change for the engine is in the lock file, just like the normal package.json?

Reply:
@lance Also revert the package-lock, should I update the lock file that only the change for the engine is in the lock file, just like the normal package.json?